src/Dfe.PlanTech.Web/Program.cs

```diff
@@ -14,9 +14,9 @@
 builder.Services.AddControllersWithViews();
 builder.Services.AddGovUkFrontend();
 
-builder.Configuration.AddAzureKeyVault(
```

Comment:
Revert this please buddy

Reply:
Sorted: https://github.com/DFE-Digital/plan-technology-for-your-school/pull/99/commits/42a0ce32ea903c033446508f714713dae167a995